modules/installation-and-upgrade/pages/proxy-container-installation.adoc

```diff
@@ -120,6 +120,11 @@ In this file it is possible to modify the tag to use for the container images:
 
 - [literal]``TAG``=[path]``latest``
 
+[IMPORTANT]
+====
+Do not change the [literal]``TAG`` value without knowing what you are doing or having been instructed to do it.
```

Comment:
We must say something such as "Changing the TAG setting is dangerous and can cause a non-functional system."

What's about the support status?

Maybe, we can add: "Only change the setting if SUSE asks you to change it."

Reply:
The `EXTRA_PODS_ARGS` may be required for people using avahi, which I doubt customers do. This can be indeed dangerous too and probably we should target the whole file for the warning.

For the support status, remember that this only applies to SUSE Manager, not Uyuni. And we cannot tell the user they won't be supported at all if they change that file: there could be valid reasons to do so (using an internal registry for instance).